nexuse2e-webapp/src/main/frontend/src/app/data/data.service.ts

```diff
@@ -74,30 +73,30 @@ export class DataService {
     return httpParams;
   }
 
-  getMessages(pageIndex: number, itemsPerPage: number, activeFilters: ActiveFilter[]): Promise<Message[]> {
+  getMessages(pageIndex: number, itemsPerPage: number, activeFilters: { [fieldName: string]: string | DateRange | undefined }): Promise<Message[]> {
     let params = DataService.buildFilterParams(activeFilters);
     params = params.append("pageIndex", String(pageIndex));
     params = params.append("itemsPerPage", String(itemsPerPage));
     return this.get<Message[]>("/messages", false, params);
   }
 
-  getMessagesCount(activeFilters: ActiveFilter[] = []): Promise<number> {
+  getMessagesCount(activeFilters: { [fieldName: string]: string | DateRange | undefined } = {}): Promise<number> {
```

Comment:
Should we introduce a type for  `{ [fieldName: string]: string | DateRange | undefined }`

Reply:
Done